src/app/login/login.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component } from '@angular/core';
+import { AuthService } from './services/auth.service';
+import { Router } from '@angular/router';
+
+import { TokenStorageService } from './services/token-storage.service';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss'],
+})
+export class LoginComponent {
+  form: any = {
+    username: null,
+    password: null,
+  };
```

Comment:
Please create type or interface and don't use 'any'. Good practice is avoid using any

Reply:
Done.